fix: Update CLI README.md (#212)
Lightprotocol/light-protocol: cli/README.md

It also provides utilities for local development, such as running a local Solana-test-validator with all necessary Light accounts and pre-loaded programs. (```light test-validator```).

You can also execute common user actions such as shielding and sending private transfers. (```light shield```, ```light transfer```, ```light unshield```).

For the full list of available commands, see below:

### Commands

- `help`: Display help information.
- `account`: Get the current account details
- `airdrop`: Perform a native Solana or SPL airdrop to a user.
- `balance`: Retrieve the balance, inbox balance, or utxos for the user.
- `config`: Update the configuration values.
- `history`: Retrieve transaction history for the user.
- `accept-utxos`: Merge multiple utxos into a single UTXO.
- `shield`: Shield tokens for a user.
  - `shield:sol`: Shield sol for a user.
  - `shield:spl`: Shield spl tokens for a user.
- `transfer`: Transfer tokens to a recipient.
- `unshield`: Unshield tokens for a user.
  - `unshield:sol`: Unshield sol for a user.
  - `unshield:spl`: Unshield spl tokens for a user.

> - `test-validator`: starts a solana-test-validator and performs all setup tasks. Use in a separate terminal in stead of solana-test-validator

Reviewer: tiny typo: _instead_

> - `test-validator`: Starts a solana-test-validator and performs all setup tasks. Use in a separate terminal instead of solana-test-validator.

Reviewer 2: ``
```suggestion
- `test-validator`: Starts a solana-test-validator and with an initialized light environment. Use in a separate terminal instead of solana-test-validator.
```

- merkleTree:
  - `mt:authority`: Initialize, set, or get the Merkle Tree Authority.
  - `mt:configuration`: Update the configuration of the Merkle Tree NFTs, permissionless SPL tokens, and lock duration.
  - `mt:initialize`: Initialize the Merkle Tree.
  - `mt:pool`: Register a new pool type [default, spl, sol].
  - `mt:verifier`: Register a new verifier for a Merkle Tree.

- psp:
  - `psp:init`: Initialize, set, or get the Merkle Tree Authority
  - `psp:build`: Update the configuration of the Merkle Tree NFTs, permissionless SPL tokens, and lock duration.
  - `psp:test`: Perform the PSP tests.

## PSP Guide

You can find a comprehensive tutorial for building a custom Private Solana Program here: https://docs.lightprotocol.com/getting-started/creating-a-custom-psp

## License

@lightprotocol/cli is released under the GNU General Public License v3.0. See the LICENSE file for more details.

## Contact

> If you have questions, suggestions, or feedback, join the developer community on Discord: https://discord.gg/J3KvDfZpyp, or reach out at hello[at]lightprotocol[dot]com.

Reviewer 3: ```suggestion
If you have questions, suggestions, or feedback, join the developer community on [Discord](https://discord.gg/J3KvDfZpyp), or reach out at hello[at]lightprotocol[dot]com.
```

Author: thx!
